src/db/params.rs

```diff
@@ -71,16 +70,16 @@ bso_data! {
     DeleteBso {},
     GetBso {},
     PutBso {
-        sortindex: Option<i64>,
+        sortindex: Option<i32>,
```

Comment:
The API docs define `sortindex` as max 9 digits, hence I scaled this down to `i32` everywhere.

Ref: https://mozilla-services.readthedocs.io/en/latest/storage/apis-1.5.html#basic-storage-object